docs/site/DEVELOPING.md

```diff
@@ -126,6 +127,46 @@ npm script `lint:fix`.
 npm run lint:fix
 ```
 
+## Working with dependencies
+
+We use npm's
+[package-lock feature](https://docs.npmjs.com/files/package-lock.json) to speed
+up our development workflow and CI builds.
+
+When installing dependencies in individual packages, `lerna bootstrap` calls
+`npm ci` to install (deep) dependencies as specified in `package-lock.json`
+file.
```

Comment:
@raymondfeng If I understood your comments on Slack correctly, you are saying that `lerna boostrap` is not using CI mode by default? Should we expand this paragraph to explain this in more details?

Reply:
Right. See https://github.com/lerna/lerna/tree/master/commands/bootstrap#--no-ci.

> When using the default --npm-client, lerna bootstrap will call npm ci instead of npm install in **CI** environments.

I debugged the issue before and confirmed that the `ci` flag is calculated using `isCI` module if not set.